### PR TITLE
Add express example

### PR DESCRIPTION
**Pull request by xWTF**
## Changes
This PR simply adds a working express example to `README.md`.

The example use `express.d.ts` to define `Request.ws()`, and provides a working websocket+http server snippet.

### CODE REVIEW

**Comment by xWTF** [comment]
Any additional changes required? I've made the change as per your request, however there's no response for more than 1 month.

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ because express-ws is...
 pnpm i ws tinyws
 ```
 
-## Example
+## Example (tinyhttp)
```
Comment (talentlessguy) [inline README.md:38]: plz change it to this instead

```md
## Example

### tinyhttp
...

### express
...
```

**Review by talentlessguy — CHANGES_REQUESTED** [review]

**Comment by xWTF** [comment]
I believe it's 1000x faster for you to push directly into my repo instead of asking for such tiny changes. I have `Allow edits by maintainers` turned on.
Anyway, new changes pushed.

**Comment by mattfeury** [comment]
not sure if this is dead in the water, but coming from express-ws, this would be really helpful to have documented.

**Comment by talentlessguy** [comment]
@mattfeury pardon this disappeared in my notifications, could you please recreate the example?

**Pull request closed without merging**